notifications_utils/polygons.py

```diff
@@ -10,8 +12,20 @@
 from shapely.ops import unary_union
 from werkzeug.utils import cached_property
 
-wgs84_to_utm29n = Transformer.from_crs('EPSG:4326', 'EPSG:32629', always_xy=True)
-utm29n_to_wgs84 = Transformer.from_crs('EPSG:32629', 'EPSG:4326', always_xy=True)
+transformers = {
+    utm_code: {
+        'from_wgs84': Transformer.from_crs('EPSG:4326', utm_code, always_xy=True),
```

Comment:
it would be useful to add a comment explaining what wgs84 is, what do you think?

Reply:
Comment added in e2faacf

Reply:
Thank you so much, these are beautiful comments, they give all the missing context 👏🏼